app/main.py

```diff
@@ -191,73 +192,41 @@ def setup_logging(debug: Union[str, bool] = False):
 if __name__ == "__main__":
 
     time_begin = time()
-    setup_logging(os.environ.get("DEBUG", "FALSE").upper())
-
-    log.info("Starting conductor data provider API.")
 
-    # Default values for file paths
-    DD20_FILEPATH_DEFAULT = "/input/DD20.XLSM"
-    DD20_MAPPING_FILEPATH_DEFAULT = "/input/Limits_other.xlsx"
-    MRID_MAPPING_FILEPATH_DEFAULT = "/input/seg_line_mrid_PROD.csv"
-
-    # Refresh rate for checking if new data is available from files
-    REFRESH_RATE_API_INPUT = 60
-
-    # Load file paths from env vars - or use defaults
-    log.info("Loading environment variables.")
+    log.info("Loading environment variables in to DD20Settings.")
     try:
-        dd20_filepath = os.environ.get("DD20_FILEPATH", DD20_FILEPATH_DEFAULT)
-        dd20_mapping_filepath = os.environ.get(
-            "DD20_MAPPING_FILEPATH", DD20_MAPPING_FILEPATH_DEFAULT
-        )
-        mrid_mapping_filepath = os.environ.get(
-            "MRID_MAPPING_FILEPATH", MRID_MAPPING_FILEPATH_DEFAULT
-        )
-    except Exception:
-        raise ValueError("Error while loading one or more file paths.")
+        for k, v in os.environ.items():
+            print(f'{k}={v}')
 
-    # Setup mock data by using test files for input, if mock flag is set true
-    if os.environ.get("USE_MOCK_DATA", "FALSE").upper() == "FALSE":
-        pass
-    elif os.environ["USE_MOCK_DATA"].upper() == "TRUE":
-        # Mock-data will overrule file paths
-        dd20_filepath = "/test-data/DD20.XLSM"
-        dd20_mapping_filepath = "/test-data/Limits_other.xlsx"
-        mrid_mapping_filepath = "/test-data/seg_line_mrid_PROD.csv"
-    else:
-        raise ValueError(
-            "'USE_MOCK_DATA' env. variable has been set to " +
-            f"'{os.environ['USE_MOCK_DATA']}'" +
-            " but must be either 'TRUE', 'FALSE' or unset."
-        )
-
-    # Load environment variables for API
-    try:
-        api_port = int(os.environ.get("API_PORT", "5000"))
-        api_dbname = os.environ.get("API_DBNAME", "CONDUCTOR_DATA").upper()
+        settings = DD20Settings()
+    except ValidationError as validation_error:
+        log.error(validation_error.errors)
+        raise validation_error
     except Exception:
-        raise ValueError("Invalid API config (PORT and/or DBNAME)")
+        raise ValueError("Error while reading application settings.")
```

Comment:
````suggestion
    except Exception as e:
        log.error("Error while reading application settings.")
        log.exception(e)
        raise e
````
I think we need to re-raise in stead of continuing and then log well before quitting

Reply:
Weird I cannot se that there are two raise after each other

Reply:
Added log statement before raise